app/views/steps/client/contact_details/edit.html.erb

```diff
@@ -10,6 +10,11 @@
     <%= step_form @form_object do |f| %>
       <%= f.govuk_phone_field :telephone_number, autocomplete: 'off', width: 'one-third', label: { size: 'm' } %>
 
+      
+      <%= f.govuk_collection_radio_buttons :correspondence_address_type, 
+                                            @form_object.choices,
+                                            :value %>
```

Comment:
I think we might want to customise the legend as it seems too big (and tag is h1):
`legend: { tag: 'span', size: 'm' }`

In reality, I think we have to take a look at the defaults set in `config/initializers/form_builder.rb` because it feels like those defaults are not working for us in this app. We can change them so the amount of times we have to tweak tags/size on some inputs is reduced.

Reply:
Shall I leave as is then and we can put up a separate PR with the correct defaults?

Reply:
Yep, lets do it separately as changing the defaults will mean we have to go through all the existing form pages and ensure they are good or make tweaks.

I think the defaults are perfect for when you have 1 question per page (usually radios) as it defaults the radios legend to be the h1 and heading of the page.
That's not the case the moment you have more than 1 element per page.